README.md

```diff
@@ -30,13 +30,31 @@ The contents of this repository are distributed via 3 different NPM packages:
 
 For more information on the individual packages, please read the respective `README.md`.
 
-## Audit
+## Audits
 
-The core smart contracts have been audited by [Halborn](https://www.halborn.com/).
+Our smart contracts undergo regular audits.
```

Comment:
if you accept my suggestion on https://github.com/aragon/osx/pull/511
remember to change it here if needed